robot_ws/src/person_detection_robot/launch/monitoring.launch

```diff
@@ -14,9 +14,9 @@
 
   <!-- Enable CloudWatch logging, send /rosout to CW Logs -->
   <arg name="log_group_name" default="robomaker_person_detection_example" />
-  <node pkg="cloudwatch_logger" type="cloudwatch_logger" name="cloudwatch_logger">
-    <rosparam command="load" file="$(find person_detection_robot)/config/cloudwatch_logs_config.yaml" />
-    <rosparam if="$(eval aws_region != '')" param="/cloudwatch_logger/aws_client_configuration/region" subst_value="true">$(arg aws_region)</rosparam>
-    <rosparam if="$(eval launch_id != '')" param="/cloudwatch_logger/log_group_name" subst_value="true">$(arg log_group_name)-$(arg launch_id)</rosparam>
-  </node>
+  <include file="$(find cloudwatch_logger)/launch/cloudwatch_logger.launch">
+    <arg name="config_file" value="$(find person_detection_robot)/config/cloudwatch_logs_config.yaml" />
+  </include>
+  <rosparam if="$(eval aws_region != '')" param="/cloudwatch_logger/aws_client_configuration/region" subst_value="true">$(arg aws_region)</rosparam>
+  <rosparam if="$(eval launch_id != '')" param="/cloudwatch_logger/log_group_name" subst_value="true">$(arg log_group_name)-$(arg launch_id)</rosparam>
```

Comment:
It feels we should attempt to put these parameters into the CSI .launch files. As-is, they make an assumption on the namespace `/cloudwatch_logger`, which was previously explicit in this file via `<node>`.  The same goes for kinesis' node name. I'll add a comment there. 

Thoughts?

Reply:
So region and launchid will be arg's inside the CSI .launch file and passed as arg's inside the `<include>` here? I think that makes sense.

Another alternative is to add `<arg name="node_name" value="cloudwatch_logger">` inside the `<include>` as setting the `node_name` is already an option [here ](https://github.com/aws-robotics/cloudwatchlogs-ros1/blob/master/cloudwatch_logger/launch/cloudwatch_logger.launch#L3)

Reply:
The alternative is preferred, specify the node name as an argument into the CSI launch files. (The CSI can have a default node name too.) This avoids putting the launchId into the CSI which is specific to RoboMaker 1-clicks.